src/screens/DiscoveryScreenV2/elements/Search.js

```diff
@@ -301,7 +303,7 @@ const styles = StyleSheet.create({
   },
   animatedViewContainer: (hideBackIcon) => ({
     flexDirection: 'row',
-    backgroundColor: 'white',
+    backgroundColor: COLORS.almostBlack,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The background color of the `animatedViewContainer` style has been changed from white to almostBlack. This will make the container's background almost black. Make sure this change aligns with the overall design and doesn't affect the visibility of any components within this container.
```diff
- backgroundColor: 'white'
+ backgroundColor: COLORS.almostBlack
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->